google/ggrc-core
# GGRC-8368 Remove ability map Issue to Assessment

# Issue description

User should not have ability to map Issue to Assessment

# Steps to test the changes

1. Create Assessment.
2. Create Issue and map it to Assessment.
3. Bad Request error should be raised.

# Solution description

1. Add hook forbid_map_issue_to_assessment to file src/ggrc/models/hooks/relationship.py.
2. Covered changes by some tests.

# Sanity checklist




# PR Review checklist
